angular/_typography.scss

```diff
@@ -20,6 +20,10 @@
         margin-bottom: 0;
     }
 
+    .mat-body-2 {
+        font-weight: 600;
+    }
+
```

Comment:
body2 should be regular. subtitle2 is semibold.

Reply:
https://pxblue.github.io/style/typography#type-scale

Reply:
From the chart, subtitle-2 should be 14dp.  

There is no subtitle-2 in angular material's typography.  The closest thing name-wise is `subheading-2`, but it doesn't have the correct font-size.  

mat-subheading-2:     font: 400 1rem/1.75 "Open Sans", Arial, Helvetica, sans-serif;
mat-body-2:      font: 500 0.875rem/1.43 "Open Sans", Arial, Helvetica, sans-serif;

body-2 has the correct font size so I adjusted the font-weight here instead.

Reply:
Angular material's typography naming convention does not match at all against React's.

Reply:
I checked with @huayunh on Thursday and made sure this is the correct class to update just because it's a bizarre situation.

Reply:
Yeah, Angular's typography naming convention is awful. https://material.angular.io/guide/typography 

By their definition, body2 is for "Bolder body text", which translate Angular Material's body2 to our subtitle2.

Evan, it would be helpful to include a table somewhere in our themes readme that explains which one maps to which — 

Angular Material's typography name | Angular Material's CSS class | What other more brilliant frameworks are calling it
--- | --- | ---
`display-4` | `.mat-display-4` | H1
... | ... | ...
`subheading-2` | `.mat-h3`, `.mat-subheading-2` | Subtitle 1 (idk)
`subheading-1` | `.mat-h4`, `.mat-subheading-1` | Subtitle 2 (idk)

etc

Reply:
That might be a useful update for the typography table on Doc-it

Reply:
Making a table like this would require some additional time.  I've made a feature request for this.
https://github.com/pxblue/themes/issues/149

Angular material doesn't even include the font sizes on their typography spec page, so you'd need to either go find the style in their theming or just render each class and inspect to find the style.   Either way it takes some time to do.

Reply:
I don't know for sure, I would doublecheck they match against what is rendered in React.